llumnix/llumlet/migration_coordinator.py

```diff
@@ -50,65 +48,110 @@ def __init__(self,
         self.max_stages = max_stages
         self.backend_engine = backend_engine
 
-    async def migrate_out_onestage(self, migrate_in_ray_actor: "ray.actor.ActorHandle",  migrate_out_request: LlumnixRequest, ) -> "MigrationStatus":
-        """one-stage live migration until last stage
+    async def migrate_out_running_request(self,
```

Comment:
Is it possible to integrate the logic for migrating running and waiting requests> In the future, we may also need to migrate swapped requests, so maintaining overall indirectness and extensibility is important.

Reply:
Migrating running requests and waiting requests have different stages of migration, the running requests are migrated in multiple stages and the waiting requests are migrated in one stage, and there are other slight differences also. It is hard to integrate them together in a elegant manner. So I think there is no need to over-design currently.